livekit-ffi/src/server/requests.rs

```diff
@@ -231,6 +231,70 @@ fn on_create_audio_track(
     })
 }
 
+fn on_local_track_mute(
+    server: &'static FfiServer,
+    request: proto::LocalTrackMuteRequest,
+) -> FfiResult<proto::LocalTrackMuteResponse> {
+    let ffi_track = server.retrieve_handle::<FfiTrack>(request.track_handle)?.clone();
+    
+    let mut muted = false;
+    match ffi_track.track {
+        Track::LocalAudio(track) => {
+            if request.mute {
+                track.mute();
+            } else {
+                track.unmute();
+            }
+            muted = track.is_muted();
+        }
+        Track::LocalVideo(track) => {
+            if request.mute {
+                track.mute();
+            } else {
+                track.unmute();
+            }
+            muted = track.is_muted();
+        }
+        _ => return Err(FfiError::InvalidRequest("track is not a local track".into())),
```

Comment:
i think these errors should maybe show `track_sid` for the user

Reply:
not necessary, it will just means our implementation is incorrect